locust-metrics/README.md

```diff
@@ -1,3 +1,47 @@
 # Locust Metrics Example
 
-This example shows how to optimize the voting app using the metrics built into Locust. This is useful when tuning an application the does not have metrics yet. Locust is now run as a deployment that lasts the length of the experiment rather than as the trial pod. The trial pod interacts with the Locust API to start and stop load tests.
+This example shows how to optimize the voting app using:
+- the experiment generator (controller v1.9.0+ required)
+- the metrics built into Locust
+
+This is useful when tuning an application that does not have metrics yet.
+
+## Deploy the voting app
+
+Run
+```
+kustomize build . | kubectl apply -f -
```

Comment:
should we change this to the github repo command?
```
kustomize build github.com/thestormforge/examples/voting-webapp/application | \
  kubectl apply -f -
```

Reply:
okay the problem was that I tried to go through this example without cloning the repo, if we assume people will clone it first then ignore me